homeassistant/components/velux/cover.py

```diff
@@ -73,17 +73,19 @@ def current_cover_position(self):
 
     @property
     def device_class(self):
-        """Define this cover as either window/blind/awning/shutter."""
-        if isinstance(self.node, Window):
-            return "window"
-        if isinstance(self.node, Blind):
-            return "blind"
-        if isinstance(self.node, RollerShutter):
-            return "shutter"
+        """Define this cover as either awning, blind, garage, gate, shutter or window."""
         if isinstance(self.node, Awning):
             return "awning"
+        if isinstance(self.node, Blind):
+            return "blind"
```

Comment:
There are constants in the cover entity integration for the valid device classes. Please import and use those here.

https://github.com/home-assistant/core/blob/4dcff294c9494b7c01a7446711da1c37bfc770e6/homeassistant/components/cover/__init__.py#L43-L52